### ISSUE TITLE
UUID changes if you update and publish a different locale version of the entry

### ISSUE THREAD

**Issue by Moonlight63** [Dulajdeshan/strapi-advanced-uuid#106]
Just discovered this. I have an entry that has multiple locales on it. Was planning to use this as a unique identifier for all entries for syncing data. In testing, if I create a new entry and save, then make a change and save, everything is fine, the uuid stays the same. If I create a new locale version of the entry and save, it gets its own uuid, even though I have not enabled localization for the field, and I have disable regenerate checked, and the uuid of the default locale entry also changes. Then, if I go back to the default and make some changes and save, the uuid of the other locale changes, but on the edited default locale stays the same. If I switch back to the 2nd locale and make a change, the uuid on the default locale changes again.
In short, everything is fine if you don't have locales, but if you do, this can't be used as a consistent identifier.

**Comment by myzonjkee** [comment]
I have a collection without Internationalization enabled. When I update a collection item using the API (via the PUT method), its UUID gets regenerated 😔.

**Comment by togberodrigue** [comment]
I also have the same problem

**Comment by yunusyuksel** [comment]
Same problem here

**Comment by gabrielcosi** [comment]
same issue, is there any plan to merge the linked PR?
